settings.php

```diff
@@ -65,10 +65,17 @@
     0
 ));
 
+$settings->add(new admin_setting_configcheckbox(
+    'facetoface_enableapprovals',
```

Comment:
I know the other settings are defined in the wrong place, but we shouldn't add more in that global $CFG
can you put this in the correct location facetoface/enableapprovals - defined in config_plugins rather than global $CFG - and one day we might fix the others.

Reply:
Will update, thanks. Named that on autopilot.

Reply:
Updated the setting name.

As it was a pretty quick find/replace I also updated the namespaces of the other settings. See issue #116